ext/pgsql/pgsql.c

```diff
@@ -2322,25 +2323,26 @@ PHP_FUNCTION(pg_lo_create)
 		switch (Z_TYPE_P(oid)) {
 		case IS_STRING:
 			{
+				/* TODO: Use zend_is_numeric_string/subroutine? */
```

Comment:
No, this code is there to handle integer strings larger than zend_long. is_numeric_string would convert those to floats, which defeats the purpose of this code.

Reply:
Right, but this code is lot, so maybe make it a dedicated function in the future?

Reply:
Oh sure, we should extract the common pgsql code (later...)